src/cli/components/landingUI.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { Text } from "ink";
+import * as path from "path";
+import { isFilePresent } from "../utils/fileSystem";
+import { FileStatus, TestFile } from "../../types/exercises.types";
+
+const Landing: React.FC<TestFile> = ({ testFileName }) => {
+    if (testFileName) {
+        const defaultExercisePath = path.join(
+            process.cwd(),
+            "exercises",
+            testFileName.toLowerCase(),
+        );
+        const fileStatus: FileStatus = isFilePresent(
+            defaultExercisePath,
+            testFileName,
+        );
+        if (fileStatus === FileStatus.SUCCESS) {
+            return (
+                <Text>
+                    Hello <Text color="green">{testFileName}</Text>
+                </Text>
+            );
+        } else if (fileStatus === FileStatus.INCONCLUSIVE) {
+            return (
+                <Text>
+                    <Text color="red">{testFileName}</Text> is inconclusive,
+                    please provide a uniquely identifiable test name
+                </Text>
+            );
+        }
```

Comment:
Yeah, well, this is the core UI I just wanted something functional at this point. I may even be removing this, this was added just for demonstration

Reply:
So rustling basically name files with its process ID and hope it eventually be unique. I think maybe leave a comments on the code to show that this might be removed. 

Because it will probably get removed, it's more of a reason to refactor this into another function so to allow the ease of future changes (change to different UI or remove it completely, etc)